dropwizard-client/src/main/java/io/dropwizard/client/HttpClientBuilder.java

```diff
@@ -221,6 +221,7 @@ protected ConfiguredCloseableHttpClient createClient(
             final org.apache.http.impl.client.HttpClientBuilder builder,
             final InstrumentedHttpClientConnectionManager manager,
             final String name) {
+        builder.setConnectionManagerShared(configuration.isShareConnectionManagerEnabled());
```

Comment:
This looks good, but do you want to move this down to around line 249 so its with the rest of the builder option setting?   This one is allow alone by itself.